### PR TITLE
[TOOLS] Building process for val tools was removed from default build

### PR DESCRIPTION
**Pull request by romka0017**
If you want to build tools, you should add --tools=on for build_mfx.pl

### CODE REVIEW

**Comment by onabiull** [comment]
@adydychk could you please add -DENABLE_TOOLS=ON to CI?

**Comment by adydychk** [comment]
Ok, but it needs to make some manual checks with this parameter but without this PR.

**Comment by adydychk** [comment]
@onabiull CI updated. Updated all builders except android and clang builders.

File: CMakeLists.txt [hunk 1]
```diff
@@ -46,6 +46,7 @@ if( CMAKE_SYSTEM_NAME MATCHES Linux )
 endif()
 
 option( ENABLE_ITT "Build targets with ITT instrumentation support (requires VTune)?" OFF )
+option( ENABLE_TOOLS "Build targets with additional tools" OFF )
```
Comment (dvrogozh) [inline CMakeLists.txt]: How about the code:
```
if (ENABLE_TOOLS)
  add_subdirectory(${CMAKE_HOME_DIRECTORY}/tools)
endif
```
right after line 66 in this file below. In that way we will avoid any changes and checks in the tools/* cmake files...

File: tools/asg-hevc/CMakeLists.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+if (NOT ENABLE_TOOLS)
```
Comment (dvrogozh) [inline tools/asg-hevc/CMakeLists.txt]: I believe this change is not needed - see above comment

File: tools/bs_parser_hevc/CMakeLists.txt [hunk 3]
```diff
@@ -1,3 +1,7 @@
+if (NOT ENABLE_TOOLS)
```
Comment (dvrogozh) [inline tools/bs_parser_hevc/CMakeLists.txt]: I believe this change is not needed - see above comment

File: tools/bs_parser_hevc/tools/hevc_fei_extractor/CMakeLists.txt [hunk 4]
```diff
@@ -1,3 +1,7 @@
+if (NOT ENABLE_TOOLS)
```
Comment (dvrogozh) [inline tools/bs_parser_hevc/tools/hevc_fei_extractor/CMakeLists.txt]: I believe this change is not needed - see above comment

File: tools/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+add_subdirectory(asg-hevc)
```
Comment (dvrogozh) [inline tools/CMakeLists.txt]: Copyright header is missed. Please, add.

File: tools/CMakeLists.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+add_subdirectory(bs_parser_hevc)
+add_subdirectory(bs_parser_hevc/tools/hevc_fei_extractor)
```
Comment (dvrogozh) [inline tools/CMakeLists.txt]: add new line at and of file

File: tools/bs_parser_hevc/CMakeLists.txt [hunk 7]
```diff
@@ -7,4 +7,4 @@ file( GLOB_RECURSE sources "${CMAKE_CURRENT_SOURCE_DIR}/src/*.c" "${CMAKE_CURREN
 set( defs " -DMFX_VERSION_USE_LATEST " )
 
 make_library( bs_parser_hevc_static none static )
-set( defs "" )
+set( defs "" )
```
Comment (dvrogozh) [inline tools/bs_parser_hevc/CMakeLists.txt]: add new line at end of file

**Pull request merged**